tooling/bin/generate-installers.sh

```diff
@@ -9,6 +9,6 @@ packages_build_dir=$2
 # Installers
 ########################
 sed "s|@release_version@|${release_version}|g" ./datadog-setup.php > "${packages_build_dir}/datadog-setup.php"
-if [[ ${release_version} == "1.0.0-nightly" && -n ${CIRCLE_WORKFLOW_JOB_ID:-} ]]; then
+if echo "${release_version}" | grep -qE '\+' && [ -n ${CIRCLE_WORKFLOW_JOB_ID:-} ]; then
```

Comment:
The `[[` isn't valid `sh`, and we invoke this via `sh` in multiple places. The existing code was invalid -- there were warnings/errors in the logs. I'm not sure why they didn't outright fail, though.

Reply:
I thought I reverted that part. Guess not. I actually tried using bash everywhere, but I kept hitting more places it was using `sh`, and since it wasn't causing fatal errors (just complaints, which is weird, as `set -e` is set), I wasn't confident in going forward.

Reply:
Yes, it uses `Alpine:3.12` as a test image, which doesn't have bash.